nservicebus/msmq/distributor/index.md

```diff
@@ -85,110 +43,17 @@ For more information on monitoring, see [Performance Counters](/nservicebus/oper
-
-NOTE: In versions 4 and above the sample above is using [NServiceBus.Distributor.MSMQ NuGet](https://www.nuget.org/packages/NServiceBus.Distributor.MSMQ).
-
-
-## Worker Configuration
-
-Any NServiceBus endpoint can run as a Worker node. To activate it, create a handler for the relevant messages and ensure that the `app.config` file contains routing information for the Distributor.
-
-
-### When hosting in NServiceBus.Host.exe
-
-If hosting the endpoint with NServiceBus.Host.exe, to run as a Worker, use this command line:
-
-```dos
-NServiceBus.Host.exe NServiceBus.MSMQWorker
-```
-
-or if using a version of NServiceBus that is earlier than Version 4.3:
-
-```dos
-NServiceBus.Host.exe NServiceBus.Worker
-```
-
-Configure the name of the master node server as shown in this `app.config` example. Note the `MasterNodeConfig` section:
-
-```xml
-<configuration>
-  <configSections>
-    <!-- Other sections go here -->
-    <section name="MasterNodeConfig" 
-             type="NServiceBus.Config.MasterNodeConfig, NServiceBus.Core" />
-  </configSections>
-  <!-- Other config options go here -->
-  <MasterNodeConfig Node="MachineWhereDistributorRuns"/>
-</configuration>
-```
-
-Read about the `DistributorControlAddress` and the `DistributorDataAddress` in the [Routing with the Distributor](#routing-with-the-distributor) section.
-
-
-### When self-hosting
-
-If self-hosting the endpoint here is the code required to enlist the endpoint with a Distributor.
-
-snippet: ConfiguringWorker
-
-NOTE: In versions 4 and above the sample above is using [NServiceBus.Distributor.MSMQ NuGet](https://www.nuget.org/packages/NServiceBus.Distributor.MSMQ).
-
-Similar to self hosting, ensure the `app.config` of the Worker contains the `MasterNodeConfig` section to point to the host name where the master node (and a Distributor) are running.
-
-
-## Is enabled in endpoint
-
-For some extensibility scenarios it may be helpful to check if the endpoint is running as either a worker, a distributor, or both.
-
+## Performance
 
-### Is running as a Distributor
+For each message being processed, the Distributor performs a few additional operations: it receives a ready message from a Worker, sends the work message to the Worker and receives a ready message post processing. That means that using Distributor introduces a certain processing overhead, that is independent of how much actual work is done. Therefore the Distributor is more suitable for relatively long running units of work (high I/O like http calls, writing to disk) as opposed to very short lived units of work (a quick read from the database and dispatching a message using `Bus.Send` or  `Bus.Publish`).
 
-snippet: IsDistributorEnabled
+To get a sense of the expected performance take the regular endpoint performance and divide it by 4.
 
+If scale out small units of work is required consider splitting the handlers into smaller vertical slices of functionality and deploying them on their own endpoints.
```

Comment:
Should be *If scaling out small units of work* or *if the scale out of small units of work*